sdk/storage/azure-storage-blobs/src/blob_client.cpp

```diff
@@ -27,8 +27,8 @@ namespace Azure { namespace Storage { namespace Blobs {
   {
     auto parsedConnectionString = Details::ParseConnectionString(connectionString);
     auto blobUri = std::move(parsedConnectionString.BlobServiceUri);
-    blobUri.AppendPath(containerName, true);
-    blobUri.AppendPath(blobName, true);
+    blobUri.AppendPath(containerName);
```

Comment:
Do we have tests for all these encoding fixes?

Reply:
i'm going to add tests tomorrow

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/664